packages/surveys/src/components/general/FileInput.tsx

```diff
@@ -7,6 +8,8 @@ import { TUploadFileConfig } from "@formbricks/types/storage";
 interface FileInputProps {
   allowedFileExtensions?: TAllowedFileExtension[];
   surveyId: string | undefined;
+  languageCode: string;
```

Comment:
una volta che in (p)react i tuoi componenti sono wrappati dentro un `IntlProvider` , non serve che tu ti faccia passare `languageCode` di padre in figlio. Il contesto `i18` é gia configurato con la localizzazione corrente. I componenti `Text` o  il decoratere/HOC `withText` infatti accedono al contesto corrente e vanno a chiamare la funzione translate con i parametri corretti. forse é superfluo anche definire la tua funzione `t(key,lang)`. 
Se vuoi accedere al contesto dal componente puoi chiederlo dopo le props.. es: `export const FileInput = (props, {intl}) => {...}` in ogni caso possiamo risparmiarci il passaggio della props `languageCode`